controllers/past-grants/views/recipient-detail.njk

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/main.njk" %}
+{% from "components/hero.njk" import hero %}
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+{% from "components/promo-card/macro.njk" import promoCard %}
+{% from "components/split-nav/macro.njk" import splitNav with context %}
+{% from "./grant-results.njk" import grantResultItem with context %}
+{% from './helpers.njk' import pgLink, buildLocationList, buildOrgTypeList with context %}
+
+{% set bodyClass = 'has-static-header' %}
+
+{% block content %}
+    <main role="main" id="content">
+        <div class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+            {{ breadcrumbTrail(breadcrumbs) }}
+
+            <div class="content-sidebar">
+                <div class="content-sidebar__primary">
+                    <div class="u-constrained-wide accent--pink">
+                        <h1>{{ title }}</h1>
+
+                        <dl class="o-definition-list o-definition-list--compact project-details">
+                            <dt>Type</dt>
+                            <dd>
+                                {{ buildOrgTypeList(organisation) }}
+                            </dd>
+
+                            {% if organisation.charityNumber %}
+                                <dt>Charity number</dt>
+                                <dd>{{ organisation.charityNumber }}</dd>
+                            {% endif %}
+
+                            {% if organisation.companyNumber %}
+                                <dt>Company number</dt>
+                                <dd>{{ organisation.companyNumber }}</dd>
+                            {% endif %}
+
+                            <dt>Location</dt>
+                            <dd>
+                                {% if recipientLocalAuthorities.length === 1 %}
+                                    {% set grant = recipientGrants[0] %}
+                                    {{ buildLocationList(grant.beneficiaryLocation) }}
+                                {% else %}
+                                    Multiple locations
+                                {% endif %}
+                            </dd>
+
+                        </dl>
+
+                        <p class="u-margin-top-l">
+                            {% set orgNameSafe = organisation.name | urlencode %}
+                            <strong>
+                                <a href="https://www.google.co.uk/search?q={{ orgNameSafe }}">
+                                    Search for {{ organisation.name }} with Google
+                                </a>
+                            </strong>
+                        </p>
+
+                    </div>
+                </div>
+
+                <div class="content-sidebar__secondary u-tone-background-tint u-padded">
```

Comment:
Rather than adding `u-tone-background-tint u-padded` probably wrap this in a `card`. See related programmes on the research page for this same pattern.

Reply:
👍